.gitignore

```diff
@@ -21,7 +21,9 @@ local_compiled_js_for_test/*
 core/templates/prod/*
 webpack_bundles/*
 core/tests/.browserstack.env
-node_modules/*
+node_modules
+.angular
+dist
```

Comment:
.angular is temp folder used by angular to put logs and stuff. dist is our angular build folder output and hence these two should not be commited.

Reply:
Added comment in code.